.github/workflows/build.yml

```diff
@@ -192,6 +192,37 @@ jobs:
           path: |
             build/coreboot.rom
           retention-days: 30
+  build_mmax:
+    runs-on: ubuntu-22.04
+    container:
+      image: ghcr.io/dasharo/dasharo-sdk:v1.4.0
+      options: --user 1001
+    strategy:
+      matrix:
+        vendor: [ intel ]
+        model: [ minnowmax ]
```

Comment:
Not sure what is wrong with this line. If anything it should complain on the next one

Reply:
Added no_sb and sb prefixes, ugly, but works